spmd/checkpoint.py

```diff
@@ -18,7 +17,7 @@
 from torch.distributed._shard.sharded_tensor.api import ShardedTensor
 
 
-def get_ranks(pg: ProcessGroup) -> List[int]:
+def get_ranks(pg: c10d.ProcessGroup) -> List[int]:
```

Comment:
Curious on the reason for this change?

Reply:
oh this is sth coming from pytorch nightly changes, where type annotation seems going away for distributed.ProcessGroup, already fixed in https://github.com/pytorch/tau/pull/495, cc @kumpera